front/src/modules/ui/button/components/FloatingIconButton.tsx

```diff
@@ -62,22 +63,22 @@ const StyledButton = styled.button<
   }};
   cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'pointer')};
   display: flex;
-
   flex-direction: row;
+
   font-family: ${({ theme }) => theme.font.family};
   font-weight: ${({ theme }) => theme.font.weight.regular};
   gap: ${({ theme }) => theme.spacing(1)};
   height: ${({ size }) => (size === 'small' ? '24px' : '32px')};
   justify-content: center;
   padding: 0;
+  position: relative;
   transition: background 0.1s ease;
   white-space: nowrap;
 
   width: ${({ size }) => (size === 'small' ? '24px' : '32px')};
 
-  &:hover {
-    background: ${({ theme, disabled }) =>
-      !disabled ? theme.background.transparent.lighter : 'transparent'};
+  &:hover .floating-icon-button-hovered {
```

Comment:
Why are we using raw CSS ?

Reply:
@lucasbordeau Because `FloatingIconButton` has a custom hover effect that doesn't take the whole size of the button, if you zoom a bit on the screenshot you'll see the effect